backend/src/test/java/VehicleResourceTest.java

```diff
@@ -0,0 +1,125 @@
+import de.cae.XYFleet.authentication.XYAuthorizer;
+import org.jooq.codegen.XYFleet.tables.records.InsurancesRecord;
+import org.jooq.codegen.XYFleet.tables.records.PricingRecord;
+import org.jooq.codegen.XYFleet.tables.records.VehiclesRecord;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.sql.Date;
+
+import static de.cae.XYFleet.authentication.XYAuthorizer.*;
+import static de.cae.XYFleet.authentication.XYAuthorizer.ROLE_ADMIN;
+import static org.jooq.codegen.XYFleet.Tables.*;
+
+public class VehicleResourceTest extends EntryResourceTest {
+    public static int INSURANCEID;
```

Comment:
The convention is usually to separate all CAPS WORDS with underscores
=> INSURANCE_ID